x-pack/plugins/aiops/common/api/log_categorization/create_category_request.ts

```diff
@@ -11,7 +11,7 @@ import { createRandomSamplerWrapper } from '@kbn/ml-random-sampler-utils';
 
 import { createCategorizeQuery } from './create_categorize_query';
 
-const CATEGORY_LIMIT = 1000;
+const CATEGORY_LIMIT = 100;
```

Comment:
I assume this was reduced for speed?
I wonder if 100 is too low? I do have some examples where I get over 100 categories. Some these least popular (rarer) categories might be interesting to the user.
Perhaps this value should be configurable. Higher for log pattern analysis and lower for log rate?

Reply:
Thanks for catching this, I overlooked that this const is part of the shared files, so I'm going to revert to 1000. If it's fine for log pattern analysis it should also work out for log rate analysis, we're only doing one categorize query there too.